dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -914,6 +913,125 @@ case class BinaryOutcomeDLCClient(
 
 object BinaryOutcomeDLCClient {
 
+  def apply(
+      outcomeWin: String,
+      outcomeLose: String,
+      oraclePubKey: ECPublicKey,
+      preCommittedR: ECPublicKey,
+      isInitiator: Boolean,
+      extPrivKey: ExtPrivateKey,
+      remoteExtPubKey: ExtPublicKey,
+      input: CurrencyUnit,
+      remoteInput: CurrencyUnit,
+      fundingUtxos: Vector[BitcoinUTXOSpendingInfoSingle],
+      remoteFundingInputs: Vector[(TransactionOutPoint, TransactionOutput)],
+      winPayout: CurrencyUnit,
+      losePayout: CurrencyUnit,
```

Comment:
`winPayout` and `losePayout` can be removed because they are in the `contractInfo`

Should be removed from the `BinaryOutcomeDLCClient` constructor as well